### PR TITLE
Docker and Docker compose

### PR DESCRIPTION
**Pull request by d3vilh**
Hi Ryan,
Please accept pull request for Docker and Docker compose image.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# {{ ansible_managed }}
```
Comment (cavaliercoder) [inline docker-compose.yml]: What's this?

**Review by cavaliercoder — COMMENTED** [review]
Thanks for this. One question.

File: docker-compose.yml (hunk 1 above)
Comment (d3vilh) [reply docker-compose.yml]: I just remove it. It was a var for comment section which suppose to have version of the compose file, but somehow it was not populated. So, I just drop it.

**Review by cavaliercoder — APPROVED** [review]

**Review by cavaliercoder — APPROVED** [review]
Noting, this won't actually build rpi-exporter. It might be advantageous to have a container that includes the build environment, then runs 'make install'.

**Comment by d3vilh** [comment]
thanks a lot

**Comment by cavaliercoder** [comment]
Do you still want this merged? I'm happy to merge but see you closed it.

**Comment by d3vilh** [comment]
Lets merge

**Pull request merged**